frontend/public/locales/zh/machines.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Machine details": "机器详情",
+  "Phase": "阶段",
+  "Provider state": "供应商状态",
+  "Node": "节点",
+  "Machine role": "机器集",
+  "Instance type": "实例类型",
+  "Region": "区域",
+  "Availability zone": "可用区",
+  "Machine addresses": "机器地址",
+  "Conditions": "条件",
+  "Name": "名称",
+  "Namespace": "命名空间",
+  "Machines": "Machine",
```

Comment:
Same as above about "Machine".

Reply:
This has not been updated.

Reply:
I can see that this had been addressed in the translation files, will send the files over again just in case.

Reply:
Machines is now updated; closing out this thread.